getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -12,11 +13,15 @@ Download the Android SDK
 Download and install the Android SDK from
 https://developer.android.com/studio/
 
+- You need to run Android Studio once to complete the setup.
```

Comment:
There's no need to make everything lists, this should not start with `- `.

That's also not really accurate as you don't need Android Studio to export for Android, only the SDK.

Reply:
it defaulted me to install Android Studio, updated docs to clarify